src/main/java/com/example/demo/database/Receipt.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo.database;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.List;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "receipts")
+public class Receipt {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long receiptId;
+
+    private long customerId;
+
+    private int totalAmount;
+
+    private Date dateTime;
+
+    @OneToMany(mappedBy = "receipt", cascade = CascadeType.ALL)
+    private List<OrderItems> orderItems;
+
+    public Receipt() {
+    }
+
+    public Receipt(long customerId, List<OrderItems> orderItems) {
+        this.customerId = customerId;
+        this.orderItems = orderItems;
+        calculateTotalAmount();
```

Comment:
The total amount will be calculated only when we use this constructor. 
So if we use no-args constructor and setters, we won't have the total amount calculated.
That's why `calculateTotalAmount()` belongs to the `ReceiptService`